src/malloc/mallocng/glue.h

```diff
@@ -51,7 +51,11 @@ static inline uint64_t get_random_secret()
 #define PAGESIZE PAGE_SIZE
 #endif
 
+#if defined(__CHEERP__) && !defined(__ASMJS__)
+#define MT (0)
```

Comment:
Why only for genericjs?

Reply:
it could be for everything, but I thought that since we will add threads soon, it's better if the code compiles and works with this stuff